src/main/java/seedu/intrack/ui/InternshipCard.java

```diff
@@ -21,7 +21,7 @@ public class InternshipCard extends UiPart<Region> {
      * As a consequence, UI elements' variable names cannot be set to such keywords
      * or an exception will be thrown by JavaFX during runtime.
      *
-     * @see <a href="https://github.com/se-edu/addressbook-level4/issues/336">The issue on AddressBook level 4</a>
+     * @see <a href="https://github.com/se-edu/addressbook-level4/issues/336">The issue on InTrack level 4</a>
```

Comment:
Should this not be changed since it is referring to a GitHub issue on AddressBook and not InTrack?